egs/chime4/s5_1ch_track/cmd.sh

```diff
@@ -0,0 +1,29 @@
+# "queue.pl" uses qsub.  The options to it are
+# options to qsub.  If you have GridEngine installed,
+# change this to a queue you have access to.
+# Otherwise, use "run.pl", which will run jobs locally
+# (make sure your --num-jobs options are no more than
+# the number of cpus on your machine.
+
+#a) JHU cluster options
```

Comment:
To make cluster independent configurations feasible `queue.pl` now uses a different format. This relies on the use of a `conf/queue.config` file. See ami/s5c/conf/queue*.conf for examples.

e.g.
<pre><code>
export train_cmd="queue.pl --mem 1G"
export decode_cmd="queue.pl --mem 2G"
</code></pre>